src/Symfony/Component/PropertyInfo/Extractor/ConstructorArgumentTypeExtractorInterface.php

```diff
@@ -18,8 +18,6 @@
  * Infers the constructor argument type.
  *
  * @author Dmitrii Poddubnyi <dpoddubny@gmail.com>
- *
- * @internal
```

Comment:
is it expected to make the interface not internal anymore but to keep its methods internal ?

Reply:
No, I must've not noticed those, removed.